basic/src/test/java/org/jboss/hal/testsuite/test/configuration/remoting/endpointconfiguration/EndpointConfigurationChannelsTestCase.java

```diff
@@ -97,4 +143,13 @@ public void editMaxOutboundMessages() throws Exception {
         }
     }
 
+    @Test
+    public void editMaxOutboundMessagesInvalidValue() throws Exception {
+        final String invalidMaxOutboundMessages = "invalid_max_outbound_messages";
+        ConfigFragment config = page.getConfigFragment();
+        config.editTextAndSave(MAX_OUTBOUND_MESSAGES, invalidMaxOutboundMessages);
+        Assert.assertTrue("Validation error should be shown", config.isErrorShownInForm());
+        config.cancel();
```

Comment:
We usually use `ConfigChecker.Builder.verifyFormNotSaved()` for these scenarios, but this is OK.